### DISCUSSION TITLE
应该呼吁停止预售制度

### DISCUSSION THREAD

**Discussion by shenzhe** [WeNeedHome/SummaryOfLoanSuspension#90] [Ideas]
PS: 大家好好讨论问题，不要什么问题都是阴阳怪气的定体问，显得自己很 "明智".

**Comment by xameless** [comment]
停了只对老百姓有好处

**Reply by jianghongping** [reply]
停了会导致房地产硬着陆，大量房地产公司倒闭。

**Reply by Go-loneliness-of-man** [reply]
这么做生意不是活该倒闭么，惯着干嘛？

**Reply by syyxsxx** [reply]
> 

已经绑架中国经济了

**Reply by YeJiashen** [reply]
> > 
> 
> 已经绑架中国经济了

所以才应该壮士断腕呀

**Reply by middleHallo** [reply]
> > 
> 
> 已经绑架中国经济了

没有什么能绑架中国经济，有，那就弄死他，不然和拿脖子怼人家刀口上有什么区别

**Comment by qinmingyuan** [comment]
我认为主要问题还是资金被非法挪用

**Reply by xameless** [reply]
那是有预售后的事，预售可以盘活市场，利好银行和房地产企业，但是对普通人没有任何好处。

**Reply by jianghongping** [reply]
预售制在房地产市场蓬勃发展阶段促进了开发商的流水也加大了他们的利润，疯狂高杠杆运行，房地产市场一歇火就歇菜了

**Comment by xujiyou** [comment]
现在停止已经没有意义了，问题已经出来了

**Reply by jianghongping** [reply]
反正房地产公司已经躺平了，硬着陆了

**Comment by Arrow-King** [comment]
其实也不过就是普通人没有办法的办法了，不然还能怎么办

**Comment by C3604** [comment]
这次事件之后银行政府开发商之间会达成新的规则

**Comment by dofindme** [comment]
顺便**取消公摊**也提上日程吧

**Comment by radiateboy** [comment]
一手交钱一手交货天经地义，预售的交易模式本来没有保护业主的合法权益，业主仅凭中国商人的“诚信”二字，上百万交给了别人。他们信任开发商，信任银行，信任政府和国家法律能够保护他们的合法权益。但是这“诚信”代价太大，这个单不应该再由业主们承担，国家应该尽早完善修改现有购房交易模式，让人民合法权益受到保护。

**Comment by dccchao** [comment]
主要是监管部门的渎职   做好资金监控  管控监管账户  专款专用  哪有这么多问题啊

**Reply by jianghongping** [reply]
主要是房价涨不上去了，开发商那种高杠杆高周转模式玩不下去了。
